### PR TITLE
Japanese Translation of Capabilities

### PR DESCRIPTION
**Pull request by kzhirata**
### Description
Japanese Translation of Capabilities
- content/documentation/webdriver/capabilities/_index.ja.md
- content/documentation/webdriver/capabilities/chromium.ja.md
- content/documentation/webdriver/capabilities/firefox.ja.md
- content/documentation/webdriver/capabilities/internet_explorer.ja.md
- content/documentation/webdriver/capabilities/safari.ja.md
- content/documentation/webdriver/capabilities/shared.ja.md

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/webdriver/capabilities/firefox.ja.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: "Capabilities specific to Firefox browser"
+title: "Firefox固有のCapabiliy"
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/capabilities/firefox.ja.md]: Capability

File: website_and_docs/content/documentation/webdriver/capabilities/firefox.ja.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 linkTitle: "Firefox"
 weight: 6
 needsTranslation: true
 description: >-
-    These capabilities are specific to Firefox.
+    これらのCapabiliyはFirefox固有のものです。
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/capabilities/firefox.ja.md]: Here too!

File: website_and_docs/content/documentation/webdriver/capabilities/internet_explorer.ja.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ---
-title: "Capabilities specific to Internet Explorer browser"
+title: "InternetExplorer固有のCapabiliy"
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/capabilities/internet_explorer.ja.md]: Same here!

File: website_and_docs/content/documentation/webdriver/capabilities/safari.ja.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Capabilities specific to Safari browser"
+title: "Safari固有のCapabiliy"
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/capabilities/safari.ja.md]: Capability

File: website_and_docs/content/documentation/webdriver/capabilities/safari.ja.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
 linkTitle: "Safari"
 weight: 10
 needsTranslation: true
 description: >-
-    These capabilities are specific to Safari.
+    これらのCapabiliyはSafari固有のものです。
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/capabilities/safari.ja.md]: Same here

**Review by harsha509 — CHANGES_REQUESTED** [review]

**Comment by kzhirata** [comment]
@harsha509 Thank you for the review. I've fixed typo.

**Review by harsha509 — APPROVED** [review]
Thank you @kzhirata!

**Pull request merged**